background/background.js

```diff
@@ -532,16 +532,20 @@ const ThinBridgeTalkClient = {
     }
     console.log(`* Check patterns for ${url}`);
 
-    for (let i = 0; i < tbconfig.URLExcludePatterns.length; i++) {
-      if (wildcmp(tbconfig.URLExcludePatterns[i][0], url)) {
-        console.log(`* Match Exclude [${tbconfig.URLExcludePatterns[i][0]}]`)
+    for (let pattern of (tbconfig.URLExcludePatterns || tbconfig.Excludes)) {
+      if (Array.isArray(pattern))
+        pattern = pattern[0];
```

Comment:
I'm not clearly understand why we need this.
Is `tbconfig.Excludes` 1 dimentional array?

Reply:
This is because the current `Excludes` is flat array but the legacy `URLExcludePatterns` was nested array.
To reduce duplicated codes with similar structure, I chose to unify current and legacy codes.

Reply:
I see, thanks!